tests/test_nplinker_local.py

```diff
@@ -10,11 +11,29 @@
 # For tests on scoring/links, add them to `scoring/test_nplinker_scoring.py`.
 
 
+def get_digest(file_path):
```

Comment:
```suggestion
def get_file_hash(file_path):
```

the name `get_digest` is a bit confusing for users who are not familiar with `hashlib`